packages/@react-aria/test-utils/src/menu.ts

```diff
@@ -58,6 +58,7 @@ export class MenuTester {
     } = opts;
 
     let trigger = this.getTrigger();
+    let isDisabled = trigger.hasAttribute('disabled');
```

Comment:
technically, if the trigger is disabled, then it cannot receive focus, so line 73 is doing something bad calling `act(() => trigger.focus());`